ArduCopter/Parameters.cpp

```diff
@@ -1069,6 +1060,17 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_GROUPINFO("GUID_TIMEOUT", 46, ParametersG2, guided_timeout, 3.0),
 #endif
 
+#if MODE_RTL_ENABLED == ENABLED
+    // @Param: RTL_ALT
+    // @DisplayName: RTL Altitude
+    // @Description: The minimum alt above home the vehicle will climb to before returning. If the vehicle is flying higher than this value it will return at its current altitude.
+    // @Units: cm
+    // @Range: 200 300000
```

Comment:
Not sure what the maximum should be since it could technically be 2^31 now... I put it 3000m 

```suggestion
    // @Range: 200 300000
```